README.md

```diff
@@ -113,22 +113,84 @@ For more installation options, see [Installer Options](https://localai.io/docs/a
 Or run with docker:
 
 ### CPU only image:
+
 ```bash
-docker run -ti --name local-ai -p 8080:8080 localai/localai:latest-cpu
+docker run -ti --name local-ai -p 8080:8080 localai/localai:latest
 ```
-### Nvidia GPU:
+
+### NVIDIA GPU Images:
+
 ```bash
+# CUDA 12.0 with core features
 docker run -ti --name local-ai -p 8080:8080 --gpus all localai/localai:latest-gpu-nvidia-cuda-12
+
+# CUDA 12.0 with extra Python dependencies
+docker run -ti --name local-ai -p 8080:8080 --gpus all localai/localai:latest-gpu-nvidia-cuda-12-extras
+
+# CUDA 11.7 with core features
+docker run -ti --name local-ai -p 8080:8080 --gpus all localai/localai:latest-gpu-nvidia-cuda-11
+
+# CUDA 11.7 with extra Python dependencies
+docker run -ti --name local-ai -p 8080:8080 --gpus all localai/localai:latest-gpu-nvidia-cuda-11-extras
+
+# NVIDIA Jetson (L4T) ARM64
+docker run -ti --name local-ai -p 8080:8080 --gpus all localai/localai:latest-nvidia-l4t-arm64
 ```
-### CPU and GPU image (bigger size):
+
+### AMD GPU Images (ROCm):
+
 ```bash
-docker run -ti --name local-ai -p 8080:8080 localai/localai:latest
+# ROCm with core features
+docker run -ti --name local-ai -p 8080:8080 --device=/dev/kfd --device=/dev/dri --group-add=video localai/localai:latest-gpu-hipblas
+
+# ROCm with extra Python dependencies
+docker run -ti --name local-ai -p 8080:8080 --device=/dev/kfd --device=/dev/dri --group-add=video localai/localai:latest-gpu-hipblas-extras
+```
+
+### Intel GPU Images (oneAPI):
+
+```bash
+# Intel GPU with FP16 support
+docker run -ti --name local-ai -p 8080:8080 localai/localai:latest-gpu-intel-f16
+
+# Intel GPU with FP16 support and extra dependencies
+docker run -ti --name local-ai -p 8080:8080 localai/localai:latest-gpu-intel-f16-extras
```

Comment:
Presently this image doesn't exist on Docker hub, but we do have master-sycl-f16-extras

Reply:
I see further down some extras images are added for the other GPUs, but intel seems to be different and we have gpu-intel and sycl?

Reply:
right there is some inconsistency with intel, however the images are already pushed. let's have another round on making more consistent naming for images